six/modules/c++/six.sicd/source/GeoData.cpp

```diff
@@ -19,36 +19,113 @@
  * see <http://www.gnu.org/licenses/>.
  *
  */
-#include <memory>
-
-#include "six/sicd/GeoData.h"
-
-namespace six
-{
-namespace sicd
-{
-GeoInfo* GeoInfo::clone() const
-{
-    return new GeoInfo(*this);
-}
-
-bool GeoInfo::operator==(const GeoInfo& rhs) const
-{
-    return (name == rhs.name && geoInfos == rhs.geoInfos &&
-            desc == rhs.desc && geometryLatLon == rhs.geometryLatLon);
-}
-
-GeoData* GeoData::clone()
-{
-    return new GeoData(*this);
-}
-
-bool GeoData::operator==(const GeoData& rhs) const
-{
-    return (earthModel == rhs.earthModel && scp == rhs.scp &&
-            imageCorners == rhs.imageCorners && validData == rhs.validData
-            && geoInfos == rhs.geoInfos);
-}
-}
-}
+#include <memory>
+
+#include "six/sicd/GeoData.h"
+#include "six/sicd/ImageData.h"
+
+namespace six
+{
+namespace sicd
+{
+GeoInfo* GeoInfo::clone() const
+{
+    return new GeoInfo(*this);
+}
+
+bool GeoInfo::operator==(const GeoInfo& rhs) const
+{
+    return (name == rhs.name && geoInfos == rhs.geoInfos &&
+            desc == rhs.desc && geometryLatLon == rhs.geometryLatLon);
+}
+
+const double GeoData::ECF_THRESHOLD = 1e-2;
```

Comment:
For consistency with other files, put class constants like this one above all the other methods